Documentation/articles/introduction.md

```diff
@@ -0,0 +1,3 @@
+# Introduction
+
+help?
```

Comment:
Remove as this isn't needed, the index page already provides an introduction

Reply:
Thats me calling for help, @tomspilman is working on this page XD